src/pages/SentPage/index.jsx

```diff
@@ -0,0 +1,11 @@
+import SentList from "../../components/SentList"
+
+function SentPage() {
```

Comment:
Since all this component does is display the SentList, it's just adding unnecesssary layers to your component tree. You should remove this component, move SentList into pages, rename it to have Page on the end and then plug it directly into the route.